frontend/src/component/impact-metrics/metricsFormatters.ts

```diff
@@ -52,13 +54,9 @@ export const getSeriesLabel = (metric: Record<string, string>): string => {
 };
 
 export const formatLargeNumbers = (value: number): string => {
-    if (value >= 1000000) {
-        return `${(value / 1000000).toFixed(0)}M`;
-    }
-    if (value >= 1000) {
-        return `${(value / 1000).toFixed(0)}k`;
-    }
-    return value.toString();
+    const formatter = prettifyLargeNumber(1000, 1);
+    const result = formatter(value);
+    return result.replace(/K/g, 'k');
```

Comment:
Huh. I'm curious. Why do this replacement? Seems arbitrary? Anyway, we should be able to set this directly in the [millify function, according to the docs](https://www.npmjs.com/package/millify#options). We can either lowercase everything, or just the `k` if we want to.